config/application.rb

```diff
@@ -30,20 +18,45 @@ class Application < Rails::Application
       #{config.root}/app/workers
     )
 
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
+    # Use Rails 7 zeitwerk eager loading
+    config.eager_load_paths += %W(
+      #{config.root}/app/models/validators
+      #{config.root}/app/representers/concerns
+      #{config.root}/app/workers
+    )
+
+    # # Do not swallow errors in after_commit/after_rollback callbacks.
+    # config.active_record.raise_in_transactional_callbacks = true
+
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 7.0
 
     config.active_job.queue_adapter = :shoryuken
     config.active_job.queue_name_prefix = ENV['ANNOUNCE_RESOURCE_PREFIX'] || ENV['RAILS_ENV']
     config.active_job.queue_name_delimiter = '_'
 
-    config.assets.enabled = false
-
-    config.cache_store = :memory_store, { size: 128.megabytes }
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
+    #
+    # config.time_zone = "Central Time (US & Canada)"
+    # config.eager_load_paths << Rails.root.join("extras")
+    # health check path
+    class HealthCheckMiddleware
```

Comment:
Do we need this?  I thought the Feeder health check was just `/api/v1` or something.

Reply:
We don't have to, but I saw the same pattern in augury and exchange, so it seemed consistent.